### DISCUSSION TITLE
Does the authentication with API key not work for playground ?

### DISCUSSION THREAD

**Discussion by Shamik-07** [agno-agi/agno#1420] [Q&A] [answered]
As mentioned in https://docs.phidata.com/agent-ui#authenticate-with-phidata, i am exporting my API key, but when i am running the playground, it's still asking me to login. Is this the expected behaviour? 
I was expecting it to not ask me for any login and i could directly access http://phidata.app/playground/.

**Comment by Shamik-07** [comment]
@jacobweiss2305 / can anyone else help with this ?

**Comment by anuragts** [comment, marked as answer]
Hey @Shamik-07  currently playground require users to login in to use it. However, we have plans for the future to remove it.

**Reply by Shamik-07** [reply]
Thanks for the response.
We should update the docs as it's unclear if login is required.

**Reply by Shamik-07** [reply]
Here's the https://github.com/phidatahq/phidata-docs/pull/93
